.gitignore

```diff
@@ -2,3 +2,4 @@ _book
 node_modules
 .*
 book
+!.github
```

Comment:
What's the rationale behind this line?

Reply:
`.*` excludes all dotfiles, `!.github` is then an exception to this rule (without this one had to `git add --force` the new workflow files, though changes will then be tracked regardless of this rule)

Reply:
Oh, I see -- I somehow missed the line above. I'm not sure if blanket-ignoring dotfiles is a good idea, but this is in line with what's there right now so no reason to block merging on it. r+